types/src/v24/wallet/into.rs

```diff
@@ -141,3 +143,27 @@ impl ListUnspentItem {
         })
     }
 }
+
+impl SendAll {
+    /// Converts version specific type to a version nonspecific, more strongly typed type.
+    pub fn into_model(self) -> Result<model::SendAll, SendAllError> {
+        use SendAllError as E;
+
+        let txid = self.txid.as_ref().map(|s| s.parse()).transpose().map_err(E::Txid)?;
+
+        let hex =
+            self.hex.as_ref().map(|h| encode::deserialize_hex(h)).transpose().map_err(E::Hex)?;
```

Comment:
This line is changed in fd06f28edd0463663a21881620545ec18e796a2d but that patch is unrelated.